daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -58,10 +57,31 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 	if err != nil {
 		return nil, err
 	}
-	if err := idtools.MkdirAllAs(home, 0700, rootUID, rootGID); err != nil {
+
+	// In order to make sure that snapshots of / don't fill up disk with our
+	// old data, make ourselves a subvolume (snapshots don't cross subvolume
+	// boundaries). So create the parent and subvol separately.
+	homeParent := filepath.Dir(home)
+	if err := idtools.MkdirAllNewAs(homeParent, 0700, rootUID, rootGID); err != nil {
```

Comment:
Is creating the parent neccessary? I would expect this to happen outside this call.